### ISSUE TITLE
Clang can't compile valid code?

### ISSUE THREAD

**Issue by sabudilovskiy** [llvm/llvm-project#64396]
Why clang can't forward consteval lvalue reference to field global consteval object? 
https://godbolt.org/z/WoKoTj9Yf

**Comment by shafik** [comment]
I believe this is https://github.com/llvm/llvm-project/issues/56695

**Comment by shafik** [comment]
Yeah, I think the other bug was mislabeled, I believe this should work in C++20 but no ETA on when a fix will come.
